addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -474,6 +474,10 @@ export class Wysiwyg extends Component {
         this.odooEditor.addEventListener('contentChanged', function () {
             self.$editable.trigger('content_changed');
         });
+        this.odooEditor.addEventListener('historyStep', () => {
+            // Flag to know if the user has made at least one change.
+            this.didUserMakeChanges = true;
```

Comment:
It's a private variable so I'd prefix it with `_`. Thanks for the comment and naming though, that makes it nice and clear :+1:

Reply:
It is not a private variable as it is used in `html_field`.

Reply:
It wasn't when I reviewed.